packages/i18n/src/resources/en.json

```diff
@@ -19,6 +19,9 @@
   "Import": "Import",
   "Trash": "Trash",
   "New Page": "New Page",
+  "Write with a blank page": "Write with a blank page",
+  "New Edgeless": "New Edgeless",
+  "Draw with a blank whiteboard": "Draw with a blank whiteboard",
```

Comment:
I prefer to change the new key to something like `com.affine.write_with_a_blank_page`